larndsim/fee.py

```diff
@@ -322,9 +322,9 @@ def export_to_hdf5(event_id_list,
                     last_time_tick = time_tick
                     packets.append(TimestampPacket(timestamp=np.floor(event_start_time_list[0] * CLOCK_CYCLE * units.mus/units.s)) ) # s
                     packets[-1].chip_key = Key(io_group,0,0)
-                    packets_mc_evt.append([-1])
-                    packets_mc_trk.append([-1] * (ASSOCIATION_COUNT_TO_STORE * 2))
-                    packets_frac.append([0] * (ASSOCIATION_COUNT_TO_STORE*2))
+                    packets_mc_evt.append(np.array([-1]))
+                    packets_mc_trk.append(np.array([-1] * track_ids.shape[1]))
+                    packets_frac.append(np.array([0] * current_fractions.shape[2]))
```

Comment:
Any reason we can't just use `MAX_TRACKS_PER_PIXEL` here?

Reply:
We can. I was thinking to do it thoroughly once altogether to enhance the readability of the code

Reply:
and to reduce inconsistency

Reply:
We can start with small steps. I'd prefer an inconsistent mixture of readability and unreadability rather than consistent unreadability.